Source/WebKit/Platform/spi/ios/UIKitSPI.h

```diff
@@ -1557,6 +1557,13 @@ typedef NS_ENUM(NSUInteger, _UIScrollDeviceCategory) {
 - (instancetype)initWithFrame:(CGRect)frame pid:(pid_t)pid contextID:(uint32_t)contextID;
 @end
 
+@interface _UIVisibilityPropagationView : UIView
+@end
+
+@interface _UINonHostingVisibilityPropagationView : _UIVisibilityPropagationView
+- (instancetype)initWithFrame:(CGRect)frame pid:(pid_t)pid environmentIdentifier:(NSString *)environmentIdentifier;
+@end
+
```

Comment:
Should we ifdef this behind HAVE_NON_HOSTING_VISIBILITY_PROPAGATION_VIEW?

Reply:
> Should we ifdef this behind HAVE_NON_HOSTING_VISIBILITY_PROPAGATION_VIEW?

That is a good point. Fixed in latest patch.

Thanks for reviewing!